toolkit/Makefile

```diff
@@ -113,17 +113,21 @@ IMAGES_DIR      ?= $(OUT_DIR)/images
 # External source server
 SOURCE_URL         ?= https://cblmarinerstorage.blob.core.windows.net/sources/core
 
+# Note on order of precedence: When a variable is passed from the commandline (i.e., make PACKAGE_URL_LIST="my list"), append
+# assignments do not take affect without using 'override'. This means that all of the following PACKAGE_URL_LIST values will
+# be ignored if the user sets any value.
```

Comment:
The first sentence makes sense, but the second one is a big confusing to me. It sounds like we **WILL** ignore the default values, if the user sets anything themselves, but I think that's the opposite of what's happening.

Reply:
No, that is correct. We would need to use `override PACKAGE_URL_LIST   += https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR_ID)/prod/base/debuginfo/$(build_arch)` to append. Instead we completely ignore all of these appends if the user does `PACKAGE_URL_LIST="my list"`. The goal here is to make `USE_PREVIEW_REPO=y` have the same affect on the URL lists as it does on the repo files (i.e., uncondinally append).

Reply:
Ah, got it, I missed the fact that the default set of URLs doesn't use `override`. Thanks for the clarification!

In that case, my suggestion for the `help:var` comment was incorrect. And judging by your docs changes, the `REBUILD_TOOLCHAIN=y` part of that comment should have been `REBUILD_TOOLCHAIN=n` as well. Could you update that comment as well?